src/qoi/src/rayfire_mesh.C

```diff
@@ -119,6 +116,10 @@ namespace GRINS
 
         // add new rayfire elem to the map
         _elem_id_map[prev_elem->id()] = elem;
+
+        // add new rayfire elem ID to parent_child map
+        add_parent(elem->id());
```

Comment:
Omitting this-> is only valid C++ as long as the member function isn't coming from a templated base class.  So your options for syntax, in increasing order of preference, are:
1. Never refactor your code to use a templated base class.
2. After any such refactor, spend an hour compiling, checking syntax errors, adding "this->", and repeating.
3. Just get in the habit of using "this->" all the time, even when redundant.